pkg/manifests/manifests_test.go

```diff
@@ -686,6 +689,16 @@ ingress:
 	if storageRequestPtr.String() != "15Gi" {
 		t.Fatal("Prometheus volumeClaimTemplate not configured correctly, expected 15Gi storage request, but found", storageRequestPtr.String())
 	}
+
+	if p.Spec.RemoteWrite[0].URL != "https://test.remotewrite.com/api/write" {
+		t.Fatal("Prometheus external labels are not configured correctly")
```

Comment:
Should the error say something about remote-write instead?

Reply:
yes